### ISSUE TITLE
Roll angle calculations fails

### ISSUE THREAD

**Issue by nicHoch** [i4Ds/STIXCore#223]
with the kernels loaded automatically from the synced folder on pub099 the roll angle calculations fails:
```
    # load the test kernel
    from stixcore.data.test import test_data
    _spm = SpiceKernelManager(test_data.ephemeris.KERNELS_DIR)
    sp = Spice(_spm.get_latest_mk())
    # DOES WORK
    pos = sp.get_orientation(date=datetime(2020, 10, 7, 12), frame='SOLO_HEEQ')
    print(pos)
    # DOES NOT WORK
    try:
        pos = sp.get_orientation(date=datetime(2021, 10, 7, 12), frame='SOLO_HEEQ')
        print(pos)
    except Exception as e:
        print(e)

    # load the latest kernel on pub099
    _spm = SpiceKernelManager(Path("/data/stix/spice/kernels/"))
    sp = Spice(_spm.get_latest_mk())
    sp.meta_kernel_path
    # DOES NOT WORK
    try:
        pos = sp.get_orientation(date=datetime(2020, 10, 7, 12), frame='SOLO_HEEQ')
        print(pos)
    except Exception as e:
        print(e)

```
returns

```
<Quantity 1.10233721 deg>, <Quantity -6.59174806 deg>, <Quantity -52.53633971 deg>
Spice returns not found for function: ckgp
Spice returns not found for function: ckgp
```

**Comment by samaloney** [comment]
Just to keep track I think this issue is due to the kernels and not the code but we should probably put a try catch somewhere perhaps in the `get_orientation` function?

With fresh kernels zip downloaded from [here](http://spiftp.esac.esa.int/data/SPICE/SOLAR-ORBITER/misc/skd/SOLAR-ORBITER.zip) 

```python
spp = Spice('<path/to/unziped/kernels/mk/solo_ANC_soc-flown-mk_v107_20220411_001.tm')
spp.get_orientation(date=datetime(2020, 10, 7, 12), frame='SOLO_HEEQ')
(<Quantity 1.10233721 deg>,
 <Quantity -6.59174806 deg>,
 <Quantity -52.53633971 deg>)
```

**Comment by samaloney** [comment]
I think this can be close now @nicHoch?

**Comment by nicHoch** [comment]
fixed in: #204
